TrafficCapture/captureOffloader/src/main/java/org/opensearch/migrations/trafficcapture/StreamLifecycleManager.java

```diff
@@ -0,0 +1,9 @@
+package org.opensearch.migrations.trafficcapture;
+
+import java.util.concurrent.CompletableFuture;
+
+public abstract class StreamLifecycleManager {
```

Comment:
nit: feels like an interface

Reply:
Thanks - will change.

Reply:
I think that it was like that before I pulled the ordering out of it.